languages/csharp/exercises/concept/nullability/.docs/introduction.md

```diff
@@ -0,0 +1,14 @@
+In C# and in many other languages, the `null` literal denotes the abscence of a value.
+
+A *nullable* type is a type that allow `null` values.
+
+There are two main families of types in C#: *reference* and *value*
```

Comment:
I'm wondering if this is where we introduce this different, or if we introduce this in the exercise that introduces variables. I'm not sure which is more appropriate.

@exercism/csharp what do you think?

Reply:
maybe the concept should be introduced somewhere else, for instance, on an exercise about value/reference parameter passing?

Reply:
Yeah, I think we should probably introduce the concept elsewhere. So for now you can assume that the student will know what reference and value types are.